packages/ckeditor5-ui/src/panel/balloon/balloonpanelview.ts

```diff
@@ -1145,6 +1174,20 @@ export function generatePositions( options: {
 					...config
 				}
 			};
+		},
+
+		// ------- Hidden
+
+		viewportHidden: () => {
+			return {
+				top: OFF_THE_SCREEN_POSITION,
+				left: OFF_THE_SCREEN_POSITION,
+				name: 'noarrow',
```

Comment:
`viewportStickyNorth` uses `arrowless`. Is there a specific reason for another name here?

Reply:
Nah, i've used the different name for debugging. It was easier to recognise which one was selected by logging `position.name`